nibabel/benchmarks/bench_array_to_file.py

```diff
@@ -29,24 +29,25 @@ def bench_array_to_file():
     sys.stdout.flush()
     print_git_title('\nArray to file')
     mtime = measure('array_to_file(arr, BytesIO(), np.float32)', repeat)
-    print('%30s %6.2f' % ('Save float64 to float32', mtime))
+    fmt = '{:30s} {:6.2f}'.format
+    print(fmt('Save float64 to float32', mtime))
```

Comment:
Have you considered f-strings instead of `str.format`?
```suggestion
    s_width = 30
    f_format = "6.2f"
    print(f'{"Save float64 to float32":{s_width}}, {mtime:{f_format}}')
```
However, f-strings might be more verbose and less readable in this case.

Reply:
I did that at first, but preferred to use a format string instead. It's less boilerplate.